### ISSUE TITLE
[TESTNET BUG] ft-get-balance returns int instead of uint when called as read-only

### ISSUE THREAD

**Issue by friedger** [blockstack/stacks-blockchain#1699]
## Describe the bug
The return value of `ft-get-balance` is an int instead of an uint, e.g. something like `(ok 20)` when called as read-only

![Screenshot from 2020-06-29 01-27-23](https://user-images.githubusercontent.com/1449049/85961053-0784a800-b9a8-11ea-893d-75c45f17ea11.png)

## Steps To Reproduce
1. deploy fungible-token contract e.g. from https://github.com/friedger/clarity-js-sdk/blob/feature/jsonrcp/packages/clarity-tutorials/contracts/tokens/fungible-token.clar
1. call read-only `balance-of` ST398K1WZTBVY6FE2YEHM6HP20VSNVSSPJTW0D53M
1. see result `(ok 20) `

## Expected behavior
result should be `(ok u20)`

## Environment

using https://testnet-explorer.blockstack.org/sandbox?tab=contract-call

## Additional context
Unit tests in clarity VM pass and handle uint values.

----

If you think this is eligible for a [bug bounty](https://testnet.blockstack.org/bounties), please check the relevant boxes below:

### Critical, Launch Blocking Bugs
**Consensus critical bugs**
- [ ] Can cause a chain split
- [ ] Can cause an invalid transaction to get mined
- [ ] Can cause an invalid block to get accepted
- [ ] Can cause a node to stall

**State corruption**
- [ ] Can modify a smart contract’s data maps and data vars without a `contract-call?

**Stolen funds**
- [ ] Any address losing STX without a corresponding transfer
- [ ] Modify token balances and NFT ownership in other contracts without a `contract-call?`

**Take control and/or bring network to a halt**
- [ ] Take control and/or bring network to a halt

### Major, Launch Blocking Bugs
**Major bugs**
- [ ] Performance or correctness bugs that don’t rise to P0 level
- [ ] Stress test or DoS attacks that slow things down enough
- [ ] Resource exhaustion
- [ ] Expected functionality doesn’t work in obvious ways (important to be super specific with this wording)

### Minor, Non-launch blocking bugs
**Minor bugs**
- [x] Bugs in non-critical software (CLI, UI, etc) that doesn’t impact critical functionality

**Comment by reedrosenbluth** [comment]
On first glance I think that this isn't a bug with the Clarity VM and that the `stacks-transactions-js` library is just deserializing uints as strings that don't include the `u`. Currently diving deeper and trying to test this hypothesis...

**Comment by reedrosenbluth** [comment]
Yup, this appears to be what's happening. Will put up a PR in `stacks-transactions-js` with this change soon.
